.github/workflows/builder_bazel_slsa3.yml

```diff
@@ -0,0 +1,69 @@
+# Copyright 2023 SLSA Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+permissions: {}
+
+on:
+  workflow_call:
+    inputs:
+      rekor-log-public:
+        description: "Allow publication of your repository name on the public Rekor log"
+        required: false
+        type: boolean
+        default: false
+      targets:
+        description: "Artifacts to build"
+        required: true
+        type: string
+      flags:
+        description: "Flags for the bazel build command"
+        required: false
+        type: string
+        default: ""
+
+    outputs:
```

Comment:
Are there any other artifacts that we can glean from the bazel config and provide here in outputs?

I imagine this is pretty hard but maybe the `bazel` command is able to tell us what it's outputs are somehow?

Reply:
What do you mean by any other artifacts? Some artifacts will need runfiles along with it in order to run, and I am working on implementing that as well. Do you mean something else though?

Reply:
I meant "could we provide the created artifacts in an output somehow?" I think it would be useful to output artifact name(s) for the binaries that were created.

Reply:
Right. The binaries are uploaded as a folder of artifacts, but to support runfiles, I am making each artifact be its own folder with the binary it represents as well as the runfiles it needs to run. Perhaps some beneficial outputs would be the names of the artifact folders and the name of the binary zip.

Reply:
@mihaimaruseac

Reply:
Just to add: the runfile support will come later and I was hypothesizing future constructive outputs. Right now, the current structure is that the artifact that is uploaded is a folder called, "binaries", which is filled with the artifacts constructed from the build.

Reply:
Additional outputs will be in future runfile PR, which will include secure-upload action for binaries folder that has outputs to send back to user